client/state/index.js

```diff
@@ -258,5 +67,5 @@ export function createReduxStore( initialState = {} ) {
 		isBrowser && window.devToolsExtension && window.devToolsExtension(),
 	].filter( Boolean );
 
-	return compose( ...enhancers )( createStore )( reducer, initialState );
+	return createStore( initialReducer, initialState, compose( ...enhancers ) );
```

Comment:
what's the story behind the change here?

Reply:
It's a drive-by change that makes creating a store with enhancers a little bit nicer. Thanks to `createStore`'s third parameter. See also https://github.com/reduxjs/redux/releases/tag/v3.1.0